source/routing/raptor_solution_reader.h

```diff
@@ -53,7 +53,23 @@ struct Dominates {
     }
 };
 
-typedef ParetoFront<Journey, Dominates> Solutions;
+struct ParetoFrontLogger {
+    log4cplus::Logger logger = log4cplus::Logger::getInstance(LOG4CPLUS_TEXT("raptor"));
+
+    void at_dominated(const Journey& to_insert, const Journey& in_pool) {
+        LOG4CPLUS_TRACE(logger, "Journey dominated ! " << std::endl
```

Comment:
Minor : Just for information it's better to use `"\n"` instead of `std::endl` that is more slowly (_endl_ flushes the output buffer)

Reply:
Thanks ! I didn't know that